index.html

```diff
@@ -9,61 +9,137 @@
+                        aria-label="Search">
+                    <button class="btn btn-outline-light" type="submit">Search</button>
+                </form>
             </div>
+        </div>
+    </nav>
 
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Calculator</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Calculator/index.html" class="btn btn-primary">Visit</a>
-                </div>
+    <section>
+        <div class="container my-5">
+            <img src="Images/home.svg" class="img-fluid" alt="">
+        </div>
+    </section>
+    <section id="about" class="container-fluid mt-5">
+        <div class="row">
+            <div class="col col-6">
+                <img src="Images/about us.svg" class="aboutUsImg img-fluid" alt="">
             </div>
 
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Color Flipper</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Color Flipper/index.html" class="btn btn-primary">Visit</a>
+            <div class="col col-6">
+                <div class="aboutSection">
+                    <h2 class="aboutHeader">About Us</h2>
+                    <p class="aboutPara">This Platform is built by JavaScript Developers to make Learning Javascript
+                        easy for the newbies
+                        . This website has so many Beginner Friendly Projects which you should try your hands on. It
+                        will help you learn each and every basic concept of Javscript and you will keep a strong hold on
+                        Javscript </p>
                 </div>
             </div>
-            <div class="card m-3" style="width: 18rem;">
-                <div class="card-body">
-                    <h5 class="card-title">Counter</h5>
-                    <p class="card-text">Some quick example text to build on the card title and make up the bulk of the
-                        card's content.</p>
-                    <a href="./Counter/index.html" class="btn btn-primary">Visit</a>
+        </div>
+    </section>
+
+
+    <section id="project">
+        <div class="container my-3 text-center">
+            <h1> Awesome Javascript Projects</h1>
+        </div>
+        <div class="container border py-4">
+            <div class="row row-cols-3">
+                <div class="card m-3" style="width: 21rem;">
+                    <img class="card-img-top" src="Images/Project1.PNG" alt="Card image cap">
```

Comment:
Change image name by project title. Like Project1.PNG doesn't define the image. But note-app.PNG define it is image of note app project. 

```suggestion
                    <img class="card-img-top" src="Images/Note-app.PNG" alt="Card image cap">
```